pybossa/task_creator_helper.py

```diff
@@ -216,19 +216,19 @@ def generate_checksum(task):
                     current_app.logger.info("duplicate task checksum error parsing task payload for project %d, %s", project.id, str(e))
                     return
             elif field == "private_json__encrypted_payload":
-                encrypted_content = task.info.get("private_json__encrypted_payload")
+                encrypted_content = task_info.get("private_json__encrypted_payload")
                 content = cipher.decrypt(encrypted_content) if cipher else encrypted_content
                 content = json.loads(content)
                 task_contents.update(content)
             else:
                 task_contents[field] = value
     else:
-        # public instance has all task fields under task.info
-        task_contents = task.info
+        # public instance has all task fields under task_info
+        task_contents = task_info
 
     checksum_fields = task_contents.keys() if not dup_fields_configured else dup_fields_configured
     checksum_payload = {field:task_contents[field] for field in checksum_fields}
-    current_app.logger.info("Project %d duplicate check fields %s", task.project_id, str(list(checksum_fields)))
+    current_app.logger.info("Project %d duplicate check fields %s", project.id, str(list(checksum_fields)))
```

Comment:
Consider logging the checksum value.

Reply:
checksum value is logged [here](https://github.com/bloomberg/pybossa/pull/1025/files/b87460400c2713e41a12ccd539bb405505152eb5#diff-6f089ebc6d4f735099b74351d353800b7cbdf445efd97796eac09a207a58439cL235)

Reply:
consolidated logs to report checksum value and duplicate task id when duplicate task is found